cmd/neofs-adm/internal/modules/morph/initialize.go

```diff
@@ -363,57 +363,45 @@ loop:
 }
 
 // sendCommitteeTx creates transaction from script and sends it to RPC.
-// If sysFee is -1, it is calculated automatically. If tryGroup is false,
-// global scope is used for the signer (useful when working with native contracts).
-func (c *initializeContext) sendCommitteeTx(script []byte, sysFee int64, tryGroup bool) error {
-	cosigners := []rpcclient.SignerAccount{{
-		Signer:  c.getSigner(tryGroup),
-		Account: c.CommitteeAcc,
-	}}
-	tx, err := c.Client.CreateTxFromScript(script, c.CommitteeAcc, sysFee, 0, cosigners)
-	if err != nil {
-		return fmt.Errorf("can't create tx: %w", err)
-	}
+// Test invocation will be performed and the result will be checked. If tryGroup
+// is false, global scope is used for the signer (useful when working with native
+// contracts).
+func (c *initializeContext) sendCommitteeTx(script []byte, tryGroup bool) error {
+	sigCount := len(c.CommitteeAcc.Contract.Parameters)
+
+	signers := make([]actor.SignerAccount, 0, sigCount)
+	signer := c.getSigner(tryGroup)
+
+	for i, w := range c.Wallets {
+		if i == sigCount {
+			break
+		}
 
-	tx.Attributes = append(tx.Attributes, transaction.Attribute{Type: transaction.HighPriority})
+		acc, err := getWalletAccount(w, committeeAccountName)
+		if err != nil {
+			return fmt.Errorf("could not get %s account for %d wallet: %w",
+				committeeAccountName, i, err)
+		}
 
-	// Calculate network fee again, because tx size has changed.
-	_, accounts, err := getSigners(c.CommitteeAcc, cosigners)
-	if err != nil {
-		panic(fmt.Errorf("BUG: can't calculate network fee: %w", err))
+		signers = append(signers, actor.SignerAccount{
```

Comment:
Judging by the neo-go code this wouldn't create a valid transaction, have you tested it?
https://github.com/nspcc-dev/neo-go/blob/master/pkg/rpcclient/actor/actor.go#L138

Is it the right way to use multisignature? @roman-khimov

Reply:
Also, `localClient` needs some love, `initialize_test.go` can help here.

Reply:
> have you tested it?

yes but only with 1/1 multisig. not sure it gonna work correctly with more then one signature

Reply:
returned signing as it was done before